xen-posix/src/mini_libc.c

```diff
@@ -321,4 +323,10 @@ unsupported_function_crash(read);
 unsupported_function_crash(gmtime);
 unsupported_function_crash(strtod);
 unsupported_function_crash(rename);
-unsupported_function_crash(strerror);
+
+unsupported_function_log(size_t, fwrite, 1);
+unsupported_function_log(int, fputc, 1);
+unsupported_function_log(int, fputs, 1);
```

Comment:
I see that `sys.c` has, for example, this code to add a fallback source of randomness:

    #ifdef HAS_UNISTD
	data[n++] = getpid();
	data[n++] = getppid();
    #endif

So turning on `HAS_UNISTD` causes us to compile extra code, which is why we then need `getpid`, etc. I guess that's OK.

What causes us to need the others? Just having then appear in a header file wouldn't be enough to require an implementation - something else must have changed...